### PR TITLE
test: run integration pipeline nightly

### PR DESCRIPTION
**Pull request by smira**
Just a copy of `integration` pipeline with the same trigger as `nightly`
pipeline, so we can have two separate pipelines and two notifications
for better visibility.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2259)

### CODE REVIEW

File: hack/drone.jsonnet [hunk 1]
```diff
@@ -335,7 +335,17 @@ local integration_trigger = {
   },
 };
 
+local integration_nightly_trigger = {
```
Comment (smira) [inline hack/drone.jsonnet:338]: unfortunately trigger expressions only support `and` conditions, so we can't make pipeline run both as promote target and cron

Comment (andrewrynhard) [reply hack/drone.jsonnet:338]: Ahh that's right. :(

**Comment by smira** [comment]
> Will this run both nightlies at the same time?

I guess so, do you think that might be a problem?

**Comment by andrewrynhard** [comment]
> > Will this run both nightlies at the same time?
> 
> I guess so, do you think that might be a problem?

Well, hopefully we should be able to tolerate more than one build running at a time, but I have seen our CI struggle to do so. I plan on fixing that soon, so we can deal with it if it is a problem.

**Pull request merged**